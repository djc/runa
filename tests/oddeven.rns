def oddeven(start: int, after: int) -> iter[int]:
	i = start
	while i < after:
		if (i % 2) == 0:
			yield i + 1
		else:
			yield i - 1
		i = i + 1

def main():
	for i in oddeven(0, 5):
		print(i)
