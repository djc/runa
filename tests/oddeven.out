1
0
3
2
5
